tracer/test/test-applications/security/Samples.Security.AspNetCore5/Properties/launchSettings.json

```diff
@@ -6,12 +6,13 @@
       "environmentVariables": {
         "ASPNETCORE_ENVIRONMENT": "Development",
         "DD_APPSEC_ENABLED": "true",
+        "DD_EXPERIMENTAL_APPSEC_STANDALONE_ENABLED": "true",
         "DD_APPSEC_BLOCKING_ENABLED": "false",
-        "DD_DOTNET_TRACER_HOME": "$(SolutionDir)shared\\bin\\monitoring-home",
+        "DD_DOTNET_TRACER_HOME": "c:\\code\\dd-trace-dotnet\\shared\\bin\\monitoring-home",
```

Comment:
Probably, changes in this file were not intended to be included in the PR

Reply:
Good catch, thanks! Fixed in d266b5895